CoursesManager.UI/Database/Procedures/Registrations/spRegistrations_Edit.sql

```diff
@@ -0,0 +1,24 @@
+﻿DELIMITER $$
+CREATE DEFINER=`courses_manager`@`%` PROCEDURE `spRegistrations_Edit`(
+	IN p_id INT,
+    IN p_course_id INT,
+    IN p_student_id INT,
+    IN p_registration_date DATE,
+    IN p_payment_status TINYINT,
+    IN p_is_active TINYINT,
+    IN p_is_achieved TINYINT,
+    IN p_updated_at DATETIME
```

Comment:
Deze kun je er ook nog uithalen

Reply:
Eruit gehaald en geupdate in de database Is_achieved wel ingelaten